AcademyApp/Extensions/UIColorExtensions.swift

```diff
@@ -35,6 +35,20 @@ extension UIColor {
         return UIColor(red: 239, green: 13, blue: 51)
     }
 
+    // color for tab the tab bar shadow
+    static var black50: UIColor {
+        return UIColor(red: 0, green: 0, blue: 0, alpha: 0.5)
+    }
+
+    // returns clear color
+    static var firstColorForGradient: CGColor {
```

Comment:
Naming, what if there will be another gradient in app?
I would suggest to keep names as in design sources - red, blue, almostBlack to keep color palette instead of naming color for their usage in app

Reply:
Renamed colors for gradient layer.